tests/unit/workflows/test_workflows_tasks_matching.py

```diff
@@ -344,7 +347,6 @@ def test_fuzzy_match_returns_true_if_something_matched(mock_match, enable_fuzzy_
     assert fuzzy_match(obj, eng)
     assert 'matches' in obj.extra_data
 
-    expected = [4328]
```

Comment:
hmm we need this for the test below, no?

Reply:
it's now defined up